packages/warehouses/src/warehouseClients/BigqueryWarehouseClient.ts

```diff
@@ -297,13 +290,7 @@ export class BigqueryWarehouseClient extends WarehouseBaseClient<CreateBigqueryC
     }
 
     async getAllTables() {
-        const client = new BigQuery({
-            projectId: this.credentials.project,
-            location: this.credentials.location,
-            maxRetries: this.credentials.retries,
-            credentials: this.credentials.keyfileContents,
-        });
-        const [datasets] = await client.getDatasets();
+        const [datasets] = await this.client.getDatasets();
```

Comment:
New ticket since this doesn't enumerate all available projects: https://github.com/lightdash/lightdash/issues/11434